core/domain/exp_services_test.py

```diff
@@ -872,6 +881,11 @@ def test_can_load_yaml_with_stripped_audio_translations(self):
 
 class GetImageFilenamesFromExplorationTests(ExplorationServicesUnitTests):
 
+    EXP_ID = 'eid'
```

Comment:
Is EXP_ID used at all?

Inline the other attrs. You only use them once.

Reply:
No. Removed. Done